src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -15,6 +18,17 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public DeleteCommand parse(String args) throws ParseException {
+        if (args.trim().length() == 0 || args.trim().equals("")) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
+        }
+
+        String[] splitBySpace = args.split(" ");
+        boolean doIndicesContainWords = checkIndicesInputContainsWords(splitBySpace);
+
+        if (doIndicesContainWords) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteCommand.MESSAGE_USAGE));
+        }
```

Comment:
If this case is index containing word, is there a reason you chose not to use MESSAGE_INVALID_INDEX in ParserUtil?

Reply:
It does not cover some scenarios e.g. `delete 1 lol`, where the issue is not that the index is invalid, but rather because the user passed in a word when they should not have. Hence, a message to explain the proper command format might be more suitable in my opinion. What do you think?